src/lib/y2network/widgets/bond_options.rb

```diff
@@ -0,0 +1,47 @@
+Yast.import "LanItems"
+require "cwm/common_widgets"
+
+module Y2Network
+  module Widgets
+    class BondOptions < CWM::ComboBox
+      def initialize(settings)
+        textdomain "network"
+        @settings = settings
+      end
+
+      def items
```

Comment:
Why not using a constant?

Reply:
good note, it can be constant as we do not refresh it.